web/src/lib/api/types/repo.ts

```diff
@@ -65,7 +67,7 @@ export interface Repo {
 
   visibility: RepoVisibility;
 
-  last_pipeline: number;
+  last_pipeline: Pipeline;
```

Comment:
Isn't this a number coming from the db. If so would rather suggest to use an additional `last_pipeline_item?: Pipeline` to avoid conflicts. Might also be an issue that the last pipeline gets unset when we are receiving updates via the sse stream.

Reply:
At least, I have not found any ref on the server for this field.